apps/server/src/modules/files-storage/controller/files-storage.controller.ts

```diff
@@ -32,13 +32,24 @@ export class FilesStorageController {
 	@Get('/download/:fileRecordId/:fileName')
 	async download(
 		@Param() params: DownloadFileParams,
-		@CurrentUser() currentUser: ICurrentUser
+		@CurrentUser() currentUser: ICurrentUser,
+		@Headers('accept') accept: string
 	): Promise<StreamableFile> {
 		const res = await this.filesStorageUC.download(currentUser.userId, params);
+
+		// TODO find out a better way to define inline or attachment
+		// This is just a straight-forward way of checking for an API request.
+		// Questions:
+		// - Should we decice by request Accept-Header?
+		// - Should we decide by response content-type?
+		// - Should we always use 'inline' over 'attachment'?
+		const isAttachment = accept ? accept.includes('application/json') : false;
+
 		// @TODO set headers ?
+
 		return new StreamableFile(res.data, {
 			type: res.contentType,
-			disposition: `attachment; filename="${params.fileName}"`,
+			disposition: `${isAttachment ? 'attachment' : 'inline'}; filename="${params.fileName}"`,
```

Comment:
I think we should use inline by default. And the customer should decide whether to preview or request a download

Reply:
Agreed. Will change that.